packages/react/test/hooks.test.tsx

```diff
@@ -0,0 +1,116 @@
+import { gql } from '@apollo/client';
+import { MockedProvider, MockedResponse } from '@apollo/client/testing';
+import { act } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+import {
+  GENERAL_SETTINGS,
+  getPostsQuery,
+  GET_URI_INFO,
+  headlessConfig,
+} from '@wpengine/headless-core';
+import React, { PropsWithChildren } from 'react';
+import { useGeneralSettings, usePosts, useUriInfo } from '../src/hooks';
```

Comment:
Looks like `usePost` and `getPostsQuery` are not used, I'm getting a TS compile error for them.

Reply:
nice catch 👍  Weird, TS was compiling fine for me. I've just pushed an update